database/migrations/20190220121107_add.js

```diff
@@ -0,0 +1,10 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.table('users', (table) =>{
+    table.string('password').notNullable()
+  });
+};
+
+exports.down = function(knex, Promise) {
+  return knex.schema.dropTable('users');
```

Comment:
Again you shouldn't just blindly drop a table here. You should be doing the opposite of what your up function is doing in your migration. In this case, you should be dropping the password column.